docs/concepts/what-is-a-schedule.md

```diff
@@ -0,0 +1,192 @@
+
+Calendar expressions are given as separate named fields.
+
+For example in tctl, they can be provided as JSON:
+
+```json
+{"year": "2022", "month": "*/3", "dayOfMonth": "1,15", "hour": "11-14"}
+```
+
+The following calendar fields are available:
+
+- `year`
+- `month`
+- `dayOfMonth`
+- `dayOfWeek`
+- `hour`
+- `minute`
+- `second`
+
+Each field may contain a comma-separated list of ranges (or `*`), and each range may include a skip value following a slash.
+
+For `month`, names of months may be used instead of integers(case-insensitive, abbreviations permitted).
+For `dayOfWeek`, day of week names may be used.
+
+Other Spec features:
+
+- **Multiple intervals/calendar expressions:** A Spec can be the union of zero or more of each of those types.
+
+**Time bounds:** An absolute start or end time (or both) can be provided. No actions will be taken before the start time or after the end time.
+
+**Exclusions:** A Spec can contain exclusions in the form of zero or more calendar expressions.
+This can be used to express things like "each Monday at noon except for holidays".
+You'll have to provide your own set of exclusions and include it in each schedule, there are no pre-defined sets.
+(Note: this isn't currently exposed in tctl or the web UI.)
+
+**Jitter:** If given, a random offset between zero and the maximum jitter is added to each action time (but bounded by the time until the next scheduled action).
+
+**Time zones:** By default, calendar-based expressions are interpreted in UTC.
+Temporal recommends using UTC to avoid various surprising properties of time zones.
+If you don't want to use UTC, you may provide the name of a time zone to interpret them in instead.
+The time zone definition will be loaded from disk on the Temporal Server Worker Service, or the fallback embedded in the binary.
+
+For even greater operational control, you may embed the contents of the time zone database file in the Schedule Spec itself.
+(Note: this isn't currently exposed in tctl or the web UI.)
+
+### Pausing
+
+A Schedule may be Paused.
+When a Schedule is Paused, it does not take any Action.
+However, you can still force manual actions, see [trigger immediately](/tctl/schedule/trigger).
+
+To assist communication among developers and operators, there's a “notes” field that can be updated on pause/unpause to store an explanation for the current state.
+
+### Limiting number of Actions
+
+A Schedule may be limited to a certain number of Actions (i.e. not trigger immediately).
+After that it will act as if it was Paused.
+
+### Policies
+
+A Schedule supports a set of Policies that enable behavior customizations.
+
+#### Overlap Policy
+
+The Overlap Policy controls what happens when it's time to start a Workflow Execution but a previously started Workflow Execution is still running.
+The following options are available:
+
+- `Skip`: **Default**.
+  Nothing happens, and the specified Action is not started.
```

Comment:
this says "Action" but the others say "Workflow Execution". probably should all say "Workflow Execution" here.

if we add other action types in the future, e.g. starting an activity or sending a signal, they will probably not be subject to the overlap policy.